packages/plugins/Wallet/src/index.ts

```diff
@@ -1,4 +1,22 @@
 // Do not export JSX from this file. It might break some runtime that does not have DOM.
-export { base } from './base.js'
+import { registerPlugin } from '@masknet/plugin-infra'
+import { base } from './base.js'
+
+registerPlugin({
+    ...base,
+    SNSAdaptor: {
+        load: () => import('./SNSAdaptor/index.js'),
+        hotModuleReload: (hot) =>
+            import.meta.webpackHot &&
+            import.meta.webpackHot.accept('./SNSAdaptor', () => hot(import('./SNSAdaptor/index.js'))),
+    },
+    Dashboard: {
+        load: () => import('./Dashboard/index.js'),
+        hotModuleReload: (hot) =>
+            import.meta.webpackHot &&
+            import.meta.webpackHot.accept('./Dashboard', () => hot(import('./Dashboard/index.js'))),
+    },
+})
+
 export * from './messages.js'
-export * from './constants.js'
+export * from './SNSAdaptor/index.js'
```

Comment:
Do not export JSX from this file. It might break some runtime that does not have DOM.

Add a new entry for this package instead.